app/mailers/deploy_mailer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class DeployMailer < ApplicationMailer
+  include InlineStylesMailer
+  default from: 'deployments@invoca.net'
```

Comment:
is this a working email group? And either way, who do we want replies to go to?

Reply:
Do we want replies to go anywhere?

Reply:
I'm sure it will be rare, but it will come up. `deploy@invoca.com` is a working group, so let's use that